app/src/main/java/com/github/geohunt/app/data/network/firebase/models/FirebaseUser.kt

```diff
@@ -11,7 +11,7 @@ package com.github.geohunt.app.data.network.firebase.models
 data class FirebaseUser(
     val id: String,
     val displayName: String?,
-    val profilePictureUrl: String
+    val profilePictureUrl: String?
```

Comment:
Maybe a better way of handling the "no-profile-picture" is to provide by default an valid url in the `constructor` bellow to a replacement profile-picture

Reply:
No, I do not really like this because we are going to create useless network connections (and also, the "no profile picture" is going to be rendered as a Composable element, using an SVG icon).